Python/README.md

```diff
@@ -117,6 +117,7 @@
 - [Hailstone Sequence](cp/Hailstone_sequence.py)
 - [Next Permutation](cp/Next_Permutation.py)
 - [Minimize Amplitude After K Removals](cp/Minimize_Amplitude_After_K_Removals.py)
+- [Find_First_and_Last_Position_of_Element_in_Sorted_Array](cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py)
```

Comment:
Rewrite it as,

`- [Find First and Last Position of Element in Sorted Array](cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py)`